lua/keymap/completion.lua

```diff
@@ -50,6 +50,10 @@ function mapping.lsp(buf)
 		["n|gd"] = map_cr("Glance definitions"):with_silent():with_buffer(buf):with_desc("lsp: Preview definition"),
 		["n|gD"] = map_cr("Lspsaga goto_definition"):with_silent():with_buffer(buf):with_desc("lsp: Goto definition"),
 		["n|gh"] = map_cr("Glance references"):with_silent():with_buffer(buf):with_desc("lsp: Show reference"),
+		["n|gm"] = map_cr("Glance implementations")
+			:with_silent()
+			:with_buffer(buf)
+			:with_desc("lsp: Goto implementation"),
```

Comment:
```suggestion
			:with_desc("lsp: Show implementation"),
```

I think Glance doesn't directly lead users to the implementation.